docs/product/relay/projects.mdx

```diff
@@ -108,3 +108,17 @@ An empty list rejects all origins. Use the default `["*"]` to allow all origins.
   }
 }
 ```
+
+`config.features`
+
+Enable product features that are not yet enabled by default.
+
+```json
+{
+  "config": {
+    "features": ["organizations:session-replay"]
+  }
+}
+```
+
+A full list of features can be found [here](https://getsentry.github.io/relay/relay_dynamic_config/enum.Feature.html#variants).
```

Comment:
Are these features all off by default and need to manually be added to the config? If so, I'd specify that like this:

```suggestion
A full list of features you can enable can be found [here](https://getsentry.github.io/relay/relay_dynamic_config/enum.Feature.html#variants).
```